src/Rules/Debug/ChainedNoDebugInNamespaceRule.php

```diff
@@ -32,13 +32,17 @@ public function processNode(Node $node, Scope $scope): array
 
         if ($message = $this->message($scope, 'App')) {
             return [
-                RuleErrorBuilder::message($message)->build(),
+                RuleErrorBuilder::message($message)
+                    ->identifier('hihaho.debug.noDebugInApp')
```

Comment:
to match the rule name
```suggestion
                    ->identifier('hihaho.debug.noChainedDebugInApp')
```